src/main/java/ratpack/zipkin/internal/RatpackCurrentTraceContext.java

```diff
@@ -0,0 +1,41 @@
+package ratpack.zipkin.internal;
+
+import brave.internal.Nullable;
+import brave.propagation.CurrentTraceContext;
+import brave.propagation.Propagation;
+import brave.propagation.TraceContext;
+import org.slf4j.MDC;
+
+public final class RatpackCurrentTraceContext extends CurrentTraceContext {
+
+  private final MDCProxy mdc;
+
+  private final TraceContext.Extractor<MDCProxy> extractor = Propagation.B3_STRING.extractor(MDCProxy::get);
+  private final TraceContext.Injector<MDCProxy> injector = Propagation.B3_STRING.injector(MDCProxy::put);
```

Comment:
impact is that in your logging configuration, you would use the keys named same as headers (ex X-B3-TraceId). Some currently use lowercamel variants like traceId. changing the key format to something different than the header names might be requested in the future, but this is a reasonable default IMHO, and so little code compared to how I was thinking.